src/Web3/Eth.elm

```diff
@@ -11,13 +11,92 @@ module Web3.Eth
 -}
 
 import Web3 exposing (Error)
-import Web3.Types exposing (CallType(..))
+import Web3.Types exposing (CallType(..), Hex)
 import Web3.Eth.Types exposing (..)
-import Web3.Decoders exposing (expectInt, expectJson, expectString)
-import Web3.Eth.Decoders exposing (blockDecoder)
+import Web3.Decoders exposing (expectInt, expectBool, expectJson, expectString, expectBigInt)
+import Web3.Eth.Decoders exposing (blockDecoder, syncStatusDecoder, txObjDecoder, blockTxObjDecoder, txReceiptDecoder)
 import Web3.Eth.Encoders exposing (txParamsEncoder)
 import Json.Encode as Encode
+import Json.Decode as Decode
 import Task exposing (Task)
+import BigInt exposing (BigInt)
+
+
+getSyncing : Task Error (Maybe SyncStatus)
+getSyncing =
+    Web3.toTask
+        { func = "eth.getSyncing"
+        , args = Encode.list []
+        , expect = expectJson (Decode.maybe syncStatusDecoder)
+        , callType = Async
+        }
+
+
+
+-- Tricky one to implement, maybe ports only?
+-- isSyncing : Task Error (Maybe SyncStatus)
```

Comment:
This might need to use our subscription being used for events. And have a similar custom native API. https://github.com/ethereum/wiki/wiki/JavaScript-API#web3ethissyncing